components/login/LoginForm.tsx

```diff
@@ -134,17 +142,28 @@ export default function LoginForm() {
                                 value={email}
                                 id="username"
                                 onChange={(e) => setEmail(e.target.value)}
+                                className="pr-10"
                             />
                         </div>
-                        <div className="mb-4">
-                            <Label htmlFor="password" className="mb-2">Password</Label>
+                        <div className="mb-4 relative">
+                            <Label htmlFor="password" className="mb-2">
+                                Password
+                            </Label>
                             <Input
-                                type="password"
+                                type={showPassword ? "text" : "password"}
                                 id="password"
                                 placeholder="Password"
                                 value={password}
```

Comment:
you can pass class (for styles here ), that would be applied just for this specific input field